Sources/CardPayments/CardClient.swift

```diff
@@ -141,6 +150,42 @@ public class CardClient: NSObject {
         return url.queryItems?.first { $0.name == param }?.value
     }
 
+    private func startVaultThreeDSecureChallenge(
+        url: URL,
+        setupTokenID: String
+    ) {
```

Comment:
```suggestion
    private func startVaultThreeDSecureChallenge(url: URL, setupTokenID: String) {
```
🧶 : only if it's within the column limit. Take or leave, a bit clearer though.